src/Orleans/Configuration/ConfigUtilities.cs

```diff
@@ -79,85 +79,6 @@ internal static void ParseTelemetry(XmlElement root, TelemetryConfiguration tele
-            }
-            if (root.HasAttribute("BulkMessageLimit"))
-            {
-                config.BulkMessageLimit = ParseInt(root.GetAttribute("BulkMessageLimit"),
-                    "Invalid int value for BulkMessageLimit attribute on Tracing element for " + nodeName);
-            }
-
-            foreach (XmlNode node in root.ChildNodes)
-            {
-                var grandchild = node as XmlElement;
-                if (grandchild == null) continue;
-
-                if (grandchild.LocalName.Equals("TraceLevelOverride") && grandchild.HasAttribute("TraceLevel") && grandchild.HasAttribute("LogPrefix"))
-                {
-                    config.TraceLevelOverrides.Add(new Tuple<string, Severity>(grandchild.GetAttribute("LogPrefix"),
-                        ParseSeverity(grandchild.GetAttribute("TraceLevel"),
-                            "Invalid trace level TraceLevel attribute value on TraceLevelOverride element for " + nodeName + " prefix " +
-                            grandchild.GetAttribute("LogPrefix"))));
-                }
-                else if (grandchild.LocalName.Equals("LogConsumer"))
-                {
-                    var className = grandchild.InnerText;
-                    Assembly assembly = null;
-                    try
-                    {
-                        int pos = className.IndexOf(',');
-                        if (pos > 0)
-                        {
-                            var assemblyName = className.Substring(pos + 1).Trim();
-                            className = className.Substring(0, pos).Trim();
-                            assembly = Assembly.Load(new AssemblyName(assemblyName));
-                        }
-                        else
-                        {
-                            assembly = typeof(ConfigUtilities).GetTypeInfo().Assembly;
-                        }
-                        var pluginType = assembly.GetType(className);
-                        if (pluginType == null) throw new TypeLoadException("Cannot locate plugin class " + className + " in assembly " + assembly.FullName);
-                        var plugin = Activator.CreateInstance(pluginType);
-
-                        if (plugin is ILogConsumer)
-                        {
-                            LogManager.LogConsumers.Add(plugin as ILogConsumer);
-                        }
-                        else
-                        {
-                            throw new InvalidCastException("LogConsumer class " + className + " must implement Orleans.ILogConsumer interface");
-                        }
-                    }
-                    catch (Exception exc)
-                    {
-                        throw new TypeLoadException("Cannot load LogConsumer class " + className + " from assembly " + assembly + " - Error=" + exc);
-                    }
-                }
-            }
-
             SetTraceFileName(config, nodeName, DateTime.UtcNow);
```

Comment:
>SetTraceFileName(config, nodeName, DateTime.UtcNow); [](start = 12, length = 52)

If you aren't parsing the TraceFilePattern any more, why is this still here?

Reply:
this is for passing trace file pattern from xml config, which load tests is still using. So I'd like to conserve that in this pr so that I don't break load test. We can remove ITraceConfiguration and its infrastructure in my next pr/